google_maps/public/html/google_maps_edit.html

```diff
@@ -17,50 +17,62 @@ <h2 class="sr">{% trans "XBlock settings" %}</h2>
+                <span class="sr">{% trans "Reset iframe width to its default value" %}</span>
+              </button>
+            </div>
+          </li>
 
-        <li class="field comp-setting-entry is-set">
-          <div class="wrapper-comp-setting">
-            <label class="label setting-label" for="iframe-width-input">{{ fields.iframe_width.display_name }}:</label>
-            <input class="input setting-input edit-iframe-width" id="iframe-width-input" value="{{ self.iframe_width }}" type="number" 
-            data-default-value="{{ fields.iframe_width.default }}">
-            <button class="action setting-clear clear-iframe-width" type="button" name="setting-clear">
-              <i class="icon fa fa-undo" aria-hidden="true"></i>
-              <span class="sr">{% trans "Reset iframe width to its default value" %}</span>
-            </button>
-          </div>
-        </li>
+          <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+              <label class="label setting-label" for="iframe-height-input">{{ fields.iframe_height.display_name }}:</label>
+              <input class="input setting-input edit-iframe-height" id="iframe-height-input" value="{{ self.iframe_height }}" type="number"
+              data-default-value="{{ fields.iframe_height.default }}">
+              <button class="action setting-clear clear-iframe-height" type="button" name="setting-clear">
+                <i class="icon fa fa-undo" aria-hidden="true"></i>
+                <span class="sr">{% trans "Reset iframe height to its default value" %}</span>
+              </button>
+            </div>
+          </li>
+        </div>
 
-        <li class="field comp-setting-entry is-set">
-          <div class="wrapper-comp-setting">
-            <label class="label setting-label" for="iframe-height-input">{{ fields.iframe_height.display_name }}:</label>
-            <input class="input setting-input edit-iframe-height" id="iframe-height-input" value="{{ self.iframe_height }}" type="number" 
-            data-default-value="{{ fields.iframe_height.default }}">
-            <button class="action setting-clear clear-iframe-height" type="button" name="setting-clear">
-              <i class="icon fa fa-undo" aria-hidden="true"></i>
-              <span class="sr">{% trans "Reset iframe height to its default value" %}</span>
-            </button>
-          </div>
-        </li>
+        <div class="google-maps-preview">
+          <iframe id="google-map" class="google-map" frameborder="0"
+                  title="{% blocktrans with place_name=self.place_name %}Map showing {{ place_name }}{% endblocktrans %}"
+                  src="{{ self.embedded_google_maps_url }}"
+                  allowfullscreen></iframe>
+        </div>
       </div>
+      {% else %}
+      <p class="api-key-missing error-message">Google maps api key is missing from settings.</p>
```

Comment:
Also since we're keeping the API key input field, this message should read ``Google Maps API key is missing.``, since it can be set in the modal window.